e2e/portalicious/pages/ChangePasswordPage.ts

```diff
@@ -0,0 +1,48 @@
+import { Locator, Page } from 'playwright';
+import BasePage from './BasePage';
+
+class ChangePasswordPage extends BasePage {
+  page: Page;
+  readonly currentPassword: Locator;
+  readonly newPassword: Locator;
+  readonly confirmPassword: Locator;
+  readonly changePasswordButton: Locator;
+
+  constructor(page: Page) {
+    super(page);
+    this.page = page;
+    this.currentPassword = this.page.getByTestId('current-password');
+    this.newPassword = this.page.getByTestId('new-password');
+    this.confirmPassword = this.page.getByTestId('confirm-password');
```

Comment:
For consistency with the other portalicious tests, let's use testids only when strictly necessary.

```suggestion
    this.currentPassword = this.page.getByLabel('Current Password');
    this.newPassword = this.page.getByLabel('New Password');
    this.confirmPassword = this.page.getByLabel('Confirm Password');
```